spark/pom.xml

```diff
@@ -668,6 +668,16 @@
     </profile>
 
     <profile>
+      <id>hadoop-2.7</id>
```

Comment:
I have no opinion on whether a new Hadoop profile is NEEDED. I followed the trend of new profile per Hadoop release. I'm fine using -Phadoop-2.6, but then we should add a line in the README indicating Hadoop 2.7 users should continue using the -Phadoop-2.6 build option.